### ISSUE TITLE
Cannot create a virtual environment as it described in the README

### ISSUE THREAD

**Issue by thbeutin** [SCECcode/csep2#9]
```
tb@tokaido:/tmp$ git clone https://github.com/SCECcode/csep2
Cloning into 'csep2'...
remote: Enumerating objects: 262, done.
remote: Counting objects: 100% (262/262), done.
remote: Compressing objects: 100% (171/171), done.
remote: Total 1717 (delta 178), reused 156 (delta 91), pack-reused 1455
Receiving objects: 100% (1717/1717), 4.19 MiB | 6.34 MiB/s, done.
Resolving deltas: 100% (1244/1244), done.
tb@tokaido:/tmp$ cd csep2/
tb@tokaido:/tmp/csep2$ mkdir venv ; cd venv
tb@tokaido:/tmp/csep2/venv$ python3 -m venv csep
tb@tokaido:/tmp/csep2/venv$ cd ..
tb@tokaido:/tmp/csep2$ source venv/csep/bin/activate
(csep) tb@tokaido:/tmp/csep2$ pip3 install -r requirements.txt
Collecting python~=3.6 (from -r requirements.txt (line 1))
  Could not find a version that satisfies the requirement python~=3.6 (from -r requirements.txt (line 1)) (from versions: )
No matching distribution found for python~=3.6 (from -r requirements.txt (line 1))
(csep) tb@tokaido:/tmp/csep2$ 
```

**Comment by wsavran** [comment]
working on a fix for this. it will require reworking the project dependencies for this not widely used such as obspy and cartopy. anaconda is tested and works for installing a compatible environment.

**Comment by thbeutin** [comment]
It still failes:

`Command "/tmp/csep2/venv/csep/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-9jb28jhn/pyproj/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-86421smi-record/install-record.txt --single-version-externally-managed --compile --install-headers /tmp/csep2/venv/csep/include/site/python3.6/pyproj" failed with error code 1 in /tmp/pip-build-9jb28jhn/pyproj/`

[venv-installation.log](https://github.com/SCECcode/csep2/files/4293220/venv-installation.log)

(btw: how can i reopen this issue?)

**Comment by thbeutin** [comment]
Some of the issues (`error: invalid command 'bdist_wheel'`) could be fixed by installing python's `wheel` package, either by using the system's package manager or by `pip3`.

**Comment by wsavran** [comment]
thanks for sharing. these issues look to be with your system and other packages trying to be installed. the problem with pip is that it requires others setup.py files to be written correctly. the recommended way to install the requirements is to use the conda package manager, because it will handle the issues with binaries like you are experiencing, and you can create your build environment with a single command.

it looks like your pyproj issue is because you [don't have the correct statics for python installed on your computer](https://stackoverflow.com/questions/21530577/fatal-error-python-h-no-such-file-or-directory).

ive re-opened the issue until we can get this sorted out.

**Comment by thbeutin** [comment]
> thanks for sharing. these issues look to be with your system and other packages trying to be installed. the problem with pip is that it requires others setup.py files to be written correctly. the recommended way to install the requirements is to use the conda package manager, because it will handle the issues with binaries like you are experiencing, and you can create your build environment with a single command.

i see the point, but AFAIK we decided to support a conda-less environment as well.



> it looks like your pyproj issue is because you [don't have the correct statics for python installed on your computer](https://stackoverflow.com/questions/21530577/fatal-error-python-h-no-such-file-or-directory).

That was the - somehow expected - solution to the last problem in the log file.

I'd say we should
 - write some words about python3 development packages for the system (the machine owner should know how to install them)
 - add the line for installing the `wheel` package by using `pip3`

**Comment by wsavran** [comment]
yes, we should support a PyPI version of the package as well. but installing and managing dependences with pip can always be a pain. hopefully we can minimize it though.

do you know if wheel was installed when you installed the python development packages on linux?

**Comment by thbeutin** [comment]
> do you know if wheel was installed when you installed the python development packages on linux?

i installed `python3-wheel`, but it didn't worked, so i had to install `wheel` by using `pip3` in the activated `venv`.

**Comment by wsavran** [comment]
closed #10
